### PR TITLE
Add Hyrule Warriors Legends

### PR DESCRIPTION
**Pull request by LCypher00**
Adds Hyrule Warriors Legends.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/hyrule-warriors-legends/game.dat [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title = "Hyrule Warriors Legends"
+description = "Hyrule Warriors Legends is a Hack&Slash game with RPG elements set in the universe of The Legend of Zelda. It was developed by Omega Force the team behind Dynasty Warriors and Team Ninja responsible for Ninja gaiden ."
```
Comment (Leo626) [inline games/hyrule-warriors-legends/game.dat]: `hack & slash` and extra space before the period in the last sentence.

Comment (LCypher00) [reply games/hyrule-warriors-legends/game.dat]: regarding the issue report should I mention it in the game's description or on the wiki page ?

Comment (Schplee) [reply games/hyrule-warriors-legends/game.dat]: All github issues go underneath the `description` field, in a field called `github_issues`.

But this is all covered by the contribution's guide.

**Comment by LCypher00** [comment]
ok I kinda messed up, I deleted my for for this pr and can't modify it anymore (don't hurt me pls).
Is it ok if i create a new one or ? (i feel dumb rn)

**Comment by Leo626** [comment]
It's fine by me as long as you don't create multiple PRs for every little change.

**Comment by Leo626** [comment]
Superseded by #92

**Pull request closed without merging**